src/libs/auth.js

```diff
@@ -114,7 +114,7 @@ export const initializeAuth = _.memoize(async () => {
         fenceStatus: isSignedIn ? state.fenceStatus : {},
         // Load whether a user has input a cookie acceptance in a previous session on this system,
         // or whether they input cookie acceptance previously in this session
-        cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(user.getId(), cookiesAcceptedKey) : false,
+        cookiesAccepted: isSignedIn ? state.cookiesAccepted || getLocalPrefForUserId(user.getId(), cookiesAcceptedKey) : undefined,
```

Comment:
This now defaults to undefined, rather than false, because `undefined` and `false` have different semantic meaning.
`false` means they have explicitly refused cookies.
`undefined` means that we don't know whether they have accepted or refused cookies.

Reply:
Where does `cookiesAccepted` get set to `false`?

Reply:
I think just in that line - it'll be false if `state.cookiesAccepted` or `getLocalPrefForUserId(...)` is false

Reply:
It also gets set to false when a user clicks `reject` on the banner. We only save accept/reject for a user, not on the browser as a whole.